index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <main id="main">
+      <h1 id="title">Valentine Jeremiah</h1>
+      <h3 class="sub-title">The Best Microverse pair programers</h3>
+      <figure id="img-div">
+        <img src="./images/Picture1-1.png" alt="" id="image">
+        <h1 id="img-caption">This is Jeremiah and Valentine</h1>
+      </figure>
+      <div id="tribute-info">Jeremiah from Ghana and Valentine from Nigeria were paired in microverse admission trial to complete few coding challenges and a project. 
+        <br>
+        <br>
+        We both Choose to do the tribute project after completing 2 challenges from hackerank. It have been fun and a great learning proccess for both of us. We enjoy working with each other and will definatly will love to work together in future projects. 

```

Comment:
Consider using a `p` tag here for better semantics

Reply:
Fantastic job here, explaining the benefits of using the `p` tag. 😄 It will help the student to remember this the next time. However, in my view, there is still a need to write it like a **suggestion**, to make it sound less like an order, just like I mentioned in my previous comments. 😊